lib/tpm2.c

```diff
@@ -741,6 +763,24 @@ tool_rc tpm2_rsa_decrypt(ESYS_CONTEXT *ectx, tpm2_loaded_object *keyobj,
     return tool_rc_success;
 }
 
+tool_rc tpm2_rsa_encrypt(ESYS_CONTEXT *ectx,
+        tpm2_loaded_object *keyobj,
```

Comment:
every function argument needn't be on the newline, collapse this to character width per coding guideline.